tests/virt_autotest/reboot_and_wait_up_upgrade.pm

```diff
@@ -40,7 +41,7 @@ sub run {
     else {
         #OpenQA needs ssh way to trigger offline upgrade
         script_run("sed -i s/sshd=1/ssh=1/g /boot/grub2/grub.cfg /boot/grub/menu.lst");
-        print("Debug info for reboot_and_wait_up_upgrade: this is offline upgrade.\n");
+        print("Debug info for reboot_and_wait_up_upgrade: this is offline upgrade. Need to clean up redundant disks.\n") && clean_up_red_disks;
```

Comment:
use `diag` to print debug info

Reply:
I can change this according to your request